### PR TITLE
Remove alarm history 3 and 4

### PR DESCRIPTION
**Pull request by tronikos**
Doing this to improve #365 and avoid the request with `request_mod: 3`
No need to have that many alarm history sensors. HA keeps its own history for the 4 alarm sensors anyway. Having alarms/alerts is uncommon and 2 sensors for history should be enough.

### CODE REVIEW

File: econet_base.yaml [hunk 1]
```diff
@@ -130,20 +130,6 @@ text_sensor:
...
-    request_mod: 3
-    icon: "mdi:alert"
-    entity_category: "diagnostic"
-  - platform: econet
-    name: "Alarm History 4"
-    id: alarm_history_4
-    sensor_datapoint: ALARMH04
-    request_mod: 3
-    icon: "mdi:alert"
-    entity_category: "diagnostic"
```
Comment (barndawgie) [inline econet_base.yaml:146]: Are these useful to leave in should someone want them? E.g. comment them out or set the request_mod polling interval to none?

Comment (tronikos) [reply econet_base.yaml:145]: I don't see how these would be useful given HA has its own history anyway. I'd rather not leave commented out code. It's already easy for someone to copy the existing alarm history sensors and modify them to include more. And it's much easier to call the read service from HA.

Comment (tronikos) [reply econet_base.yaml:145]: If you just click on the sensor you will get the history, e.g. for a different string sensor:
image

**Comment by tronikos** [comment]
I'd like to get this merged to make room for an additional request for a few more sensors I'm planning on adding for #390 
I want to avoid making too many requests.

**Pull request merged**